jerry-core/parser/js/js-parser.c

```diff
@@ -1803,9 +1803,19 @@ parser_post_processing (parser_context_t *context_p) /**< context */
     byte_code_p += sizeof (cbc_uint8_arguments_t);
   }
 
+  uint16_t encoding_limit;
+  uint16_t encoding_delta;
+
   if (context_p->literal_count > CBC_MAXIMUM_SMALL_VALUE)
   {
     compiled_code_p->status_flags |= CBC_CODE_FLAGS_FULL_LITERAL_ENCODING;
+    encoding_limit = 128;
+    encoding_delta = 0x8000;
+  }
+  else
+  {
+    encoding_limit = 255;
+    encoding_delta = 0xfe01;
```

Comment:
How is this value calculated? Could we add any info on this?

Reply:
I've added some description about these numbers.